src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -487,32 +559,79 @@ bool CheckReadOnlyParamsForAudio(
     const auto& equalizer_settings =
         module_type_params[message_params::kEqualizerSettings];
     auto it = equalizer_settings.asArray()->begin();
+
     for (; it != equalizer_settings.asArray()->end(); ++it) {
       if (it->keyExists(message_params::kChannelName)) {
-        LOG4CXX_DEBUG(logger_, " READ ONLY parameter: ");
+        LOG4CXX_DEBUG(logger_, "READ ONLY parameter. ChannelName = " << (*it)[message_params::kChannelName].asString());
         return true;
       }
     }
   }
+
+  return false;
+}
+
+bool CheckReadOnlyParamsForLight(
+    const smart_objects::SmartObject& module_type_params) {
+  if (module_type_params.keyExists(message_params::kLightState)) {
+    const auto& light_state = module_type_params[message_params::kLightState];
+    auto it = light_state.asArray()->begin();
+
+    for (; it != light_state.asArray()->end(); ++it) {
+        if (it->keyExists(message_params::kStatus)) {
+          const mobile_apis::LightStatus::eType light_status =
+              static_cast<mobile_apis::LightStatus::eType>(
+                  (*it)[message_params::kStatus].asUInt());
+
+          if (helpers::Compare<mobile_apis::LightStatus::eType,
+                               helpers::EQ,
+                               helpers::ONE>(
+                  light_status,
+                  mobile_apis::LightStatus::RAMP_UP,
+                  mobile_apis::LightStatus::RAMP_DOWN,
+                  mobile_apis::LightStatus::UNKNOWN,
+                  mobile_apis::LightStatus::INVALID)) {
+            LOG4CXX_DEBUG(logger_, "READ ONLY parameter. Status = " << (*it)[message_params::kStatus].asInt());
+            return true;
+          }
+        }
+      }
+    }
+
   return false;
 }
 
 bool SetInteriorVehicleDataRequest::AreReadOnlyParamsPresent(
-    const smart_objects::SmartObject& module_data) {
+    const smart_objects::SmartObject& module_data,
+    ModuleCapability& module_data_capabilities) {
   LOG4CXX_AUTO_TRACE(logger_);
   const smart_objects::SmartObject& module_type_params =
       ControlData(module_data);
   auto it = module_type_params.map_begin();
```

Comment:
@KVGrygoriev  initialization of 
`auto it = module_type_params.map_begin(); `
better to move exactly before it first usage